tests/unit/components/playback/controls/PopInput.spec.js

```diff
@@ -0,0 +1,222 @@
+    const storePath = `${process.env.buildDir}/store.js`;
+    NuxtStore = await import(storePath);
+  });
+
+  beforeEach(async () => {
+    store = await NuxtStore.createStore();
+  });
+
+  afterEach(() => wrapper.destroy());
+  test("When mounting popinput from the build dist file, Then it loads successfully and the props defined title is rendered", () => {
+    const propsData = {
+      title_label: "Enter  Alphanumeric  ID",
+    };
+    wrapper = shallowMount(DistComponentToTest, {
+      propsData,
+      store,
+      localVue,
+    });
+    const target_span = wrapper.find(".span__popinput-content-label");
+
+    expect(target_span.text()).toStrictEqual("Enter  Alphanumeric  ID"); // the value of &nbsp<wbr> is '\u00a0'
+  });
+  test("When the PopInput is mounted, Then it loads successfully and the props defined placeholder is rendered on input field", () => {
+    const propsData = {
+      title_label: "Enter  Alphanumeric  ID",
+      key_placeholder: "place holder",
+    };
+    wrapper = mount(ComponentToTest, {
+      propsData,
+      store,
+      localVue,
+    });
+    const target_input = wrapper.find("#input-widget");
+    expect(target_input.attributes().placeholder).toStrictEqual("place holder");
+  });
+  test("When the PopInput is mounted, Then the user enters few charters in the input, confirm that an event 'update' is emitted with entered text", async () => {
+    const propsData = {
+      title_label: "Enter  Alphanumeric  ID",
+      key_placeholder: "place holder",
+      user_key: "",
+    };
+    wrapper = mount(ComponentToTest, {
+      propsData,
+      store,
+      localVue,
+    });
+    const uuidBase62 = "2VSckkBYH2An3dqHEyfRRE"; // proper uuidcode sent
+    const input_widget = wrapper.find("#input-widget");
+    input_widget.element.value = uuidBase62;
+    await input_widget.trigger("input");
+    const parent_id_events = wrapper.emitted("update:user_key");
+    expect(parent_id_events).toHaveLength(2);
+    expect(parent_id_events).toStrictEqual([[""], ["2VSckkBYH2An3dqHEyfRRE"]]);
+    expect(wrapper.vm.input_key_action).toStrictEqual(true); // This is a computed value its a response its important to verify the value
+  });
+  test("When the popinput is mouted, Then when by default an error text is rendered provided in the prop", async () => {
+    const propsData = {
+      title_label: "Enter  Alphanumeric  ID",
+      key_placeholder: "place holder",
+      user_key: "",
+      invalid_text: "This field is required",
+    };
+    wrapper = mount(ComponentToTest, {
+      propsData,
+      store,
+      localVue,
+    });
+    const target_div = wrapper.find(
+      ".div__popinput-controls-content-input-feedback"
+    );
+    expect(target_div.text()).toStrictEqual("This field is required");
+  });
+  test("When the PopInput is mounted, Then the user enters few charters in the input, verify that the spellcheck is set to false", async () => {
```

Comment:
is there a reason you have the user type some characters in this test? it doesn't seem like it would be necessary to test whether the spellcheck is set to false, but maybe I'm missing something

Reply:
The Eslint rules ensures the "Then the" words in the test title, so to pass Eslint added the text and for any user to verify from an npm run dev the validation of spellcheck functional work, have the description 'enters few characters in the input'.  The second point PopInput itself could be modified to pass an improper prop instead of :spellcheck it can be modified to :s_pellcheck of some sort, in order to prevent such issue, included the check for the attribute 'spellcheck=false' which is sent on the prop 'input_check: false'.